services/filesstore/s3store.go

```diff
@@ -42,19 +41,42 @@ const (
 	bucketNotFound = "NoSuchBucket"
 )
 
+var (
+	imageExtensions = [7]string{".jpg", ".jpeg", ".gif", ".bmp", ".png", ".tiff", "tif"}
+	imageMimeTypes  = map[string]string{".jpg": "image/jpeg", ".jpeg": "image/jpeg", ".gif": "image/gif", ".bmp": "image/bmp", ".png": "image/png", ".tiff": "image/tiff", ".tif": "image/tif"}
+)
+
+func isFileExtImage(ext string) bool {
+	ext = strings.ToLower(ext)
+	for _, imgExt := range imageExtensions {
+		if ext == imgExt {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Does it make sense to use a map here as well?

Reply:
:facepalm: sure, just copied the previous existing approach. I'll change it.

Reply:
done